admin/views/tabs/network/general.php

```diff
@@ -50,3 +53,5 @@
 }
 
 echo '<p><strong>' . esc_html__( 'Take note:', 'wordpress-seo' ) . '</strong> ' . esc_html__( 'Privacy sensitive (FB admins and such), theme specific (title rewrite) and a few very site specific settings will not be imported to new sites.', 'wordpress-seo' ) . '</p>';
```

Comment:
Not coming from to this PR, but concatenating strings should be avoided as it makes translations difficult form some languages. This should use a `sprintf()` with placeholders for the strong tags.